test/images/resource-consumer/Dockerfile

```diff
@@ -12,7 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jess/stress
+FROM BASEIMAGE
```

Comment:
Would it be possible to set `gcr.io/google-containers/ubuntu-slim-GOARCH:0.12` here and replace GOARCH dynamically?